src/libraries/tests.proj

```diff
@@ -618,15 +618,23 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(ArchiveTests)' == 'true' and '$(TargetOS)' == 'ios' and '$(IsManualOrRollingBuild)' == 'true'">
+    <!-- Tracking issue: https://github.com/dotnet/runtime/pull/86089 -->
+    <!-- <ProjectReference Include="$(MonoProjectRoot)sample\iOS\Program.csproj"
+                        BuildInParallel="false" /> -->
+    <!-- Tracking issue: https://github.com/dotnet/runtime/issues/80911 -->
+    <!-- <ProjectReference Include="$(MonoProjectRoot)sample\iOS-NativeAOT\Program.csproj"
+                      BuildInParallel="false" /> -->
     <!-- These crash on tvOS, but do not on iOS. Run these only on the rolling builds -->
     <ProjectReference Include="$(MSBuildThisFileDirectory)System.IO.MemoryMappedFiles\tests\System.IO.MemoryMappedFiles.Tests.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)System.Runtime.Numerics\tests\System.Runtime.Numerics.Tests.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)System.Formats.Cbor\tests\System.Formats.Cbor.Tests.csproj" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(ArchiveTests)' == 'true' and '$(RunSmokeTestsOnly)' != 'true' and '$(RunGrpcTestsOnly)' != 'true' and '$(TargetOS)' == 'iossimulator'">
-    <!-- https://github.com/dotnet/runtime/issues/57666 -->
-    <!-- <ProjectReference Include="$(MonoProjectRoot)sample\iOS\Program.csproj"
+    <ProjectReference Include="$(MonoProjectRoot)sample\iOS\Program.csproj"
+                      BuildInParallel="false" />
+    <!-- Tracking issue: https://github.com/dotnet/runtime/issues/80911 -->
+    <!-- <ProjectReference Include="$(MonoProjectRoot)sample\iOS-NativeAOT\Program.csproj"
```

Comment:
Is NativeAOT functional testing going to have a separate ItemGroup?

Reply:
We could create a separate item group for the Native AOT runtime and functional tests, so I suggest we introduce it once https://github.com/dotnet/runtime/pull/86652 and https://github.com/dotnet/runtime/pull/86089 land main.

Reply:
> so I suggest we introduce it once https://github.com/dotnet/runtime/pull/86652 and https://github.com/dotnet/runtime/pull/86089 land main.

Agreed. 
Although I think adding a `'$(RuntimeFlavor)' == 'Mono'` condition would also be suitable.